src/Workspaces/CSharp/Portable/Extensions/MemberDeclarationSyntaxExtensions_GetAttributes.cs

```diff
@@ -10,6 +10,9 @@ public static SyntaxList<AttributeListSyntax> GetAttributes(this MemberDeclarati
         {
             if (member != null)
             {
+#if !CODE_STYLE
+                return member.AttributeLists;
+#else
```

Comment:
What's the reason we need to keep the old code here? Seems trouble as we evolve the syntax nodes...

Reply:
the way the code_style layer works is that it references older versions of roslyn (i believe).  so, until we update that layer to use a later version, it won't see these members.  Tagging @sharwell to shed some light here.  I'm not sure what our general engineering story is here.  For example, would we want to open a trakcing bug with this to get rid of this code once we updated the roslyn dlls being references from code_style?